subhub/subhub_dynamodb.py

```diff
@@ -30,17 +30,39 @@ class Meta:
 class SubHubAccountModel(Model):
     user_id = UnicodeAttribute(hash_key=True)
     cust_id = UnicodeAttribute(null=True)
-    origin_system = UnicodeAttribute()
+    orig_system = UnicodeAttribute()
+    customer_status = UnicodeAttribute()
 
 
 class SubHubAccount:
     def __init__(self, table_name: str, region: str, host: Optional[str] = None):
-        self.model = _create_account_model(table_name, region, host)
+        _table = table_name
+        _region = region
+        _host = host
+
+        class SubHubAccountModel(Model):
+            class Meta:
+                table_name = _table
+                region = _region
+                if _host:
+                    host = _host
+
+            user_id = UnicodeAttribute(hash_key=True)
+            cust_id = UnicodeAttribute(null=True)
+            orig_system = UnicodeAttribute()
+            customer_status = UnicodeAttribute()
+
+        self.model = SubHubAccountModel
 
     def new_user(
         self, uid: str, origin_system: str, cust_id: Optional[str] = None
     ) -> SubHubAccountModel:
-        return self.model(user_id=uid, cust_id=cust_id, origin_system=origin_system)
+        return self.model(
+            user_id=uid,
+            cust_id=cust_id,
+            orig_system=origin_system,
```

Comment:
fix

Reply:
This is blacked, so black will just do it again